docker/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  local-linguist-server:
+    build: ..
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+    expose:
+      - "5432"
+  test:
+    build: ..
+    depends_on:
+      - db
+    command: rake
```

Comment:
I'm going to make the `db` and `test` hostnames more specific.

Reply:
Good idea!